templates/download/qualcomm-iot/tabs/rb3-gen2-tab.html

```diff
@@ -25,26 +25,104 @@ <h2 class="u-hide--small">Qualcomm RB3 Gen 2 Vision Kit (QCS6490)</h2>
+        <div class="row">
+          <div class="col-6 col-medium-4 col-small-4">
+            <hr class="p-rule--muted" />
+          </div>
+          <div class="col-3 col-medium-2 col-small-2">
+            <p class="p-heading--5">Ubuntu Desktop 24.04 for QCS6490</p>
+          </div>
+          <div class="col-3 col-medium-2 col-small-2">
+            <a class="p-button--positive"
+               href="https://people.canonical.com/~platform/images/qualcomm-iot/ubuntu-24.04/ubuntu-24.04-x03/ubuntu-desktop-24.04"
+               title="Download Ubuntu Desktop 24.04 for QCS6490">Download</a>
+          </div>
+          <div class="col-6 col-medium-4 col-small-4">
+            <hr class="p-rule--muted" />
+          </div>
+          <div class="col-3 col-medium-2 col-small-2">
+            <p class="p-heading--5">Ubuntu Server 24.04 for QCS6490</p>
+          </div>
+          <div class="col-3 col-medium-2 col-small-2">
+            <a class="p-button--positive"
+               href="https://people.canonical.com/~platform/images/qualcomm-iot/ubuntu-24.04/ubuntu-24.04-x03/ubuntu-server-24.04"
+               title="Download Ubuntu Server 24.04 for QCS6490">Download</a>
+          </div>
+          <div class="col-6 col-medium-4 col-small-4">
+            <hr class="p-rule--muted" />
+          </div>
+          <div class="col-3 col-medium-2 col-small-2">
+            <p class="p-heading--5">Boot firmware</p>
+          </div>
+          <div class="col-3 col-medium-2 col-small-2">
+            <a class="p-button"
+               href="https://artifacts.codelinaro.org/ui/native/qli-ci/flashable-binaries/ubuntu-fw/QLI.1.4-Ver.1.1-ubuntu-nhlos-bins.tar.gz"
+               title="Download boot firmware">Download</a>
+          </div>
+        </div>
+      </div>
+
+      <ul class="p-list--divided">
+        <li class="p-list__item">
+          Get started with Ubuntu 24.04 for <a href="https://www.qualcomm.com/products/internet-of-things/industrial/building-enterprise/qcs6490#overview">QCS6490</a>
+        </li>
+        <li class="p-list__item">
+          Ubuntu 24.04 for QCS6490 <a href="https://docs.qualcomm.com/bundle/publicresource/topics/80-90441-1/Integrate_and_flash_software_2.html?product=1601111740057201">image installation instructions</a>
+        </li>
+        <li class="p-list__item">
+          Ubuntu 24.04 for QCS6490 <a href="https://people.canonical.com/~platform/images/qualcomm-iot/ubuntu-24.04/ubuntu-24.04-x03/Ubuntu_24.04_for_Qualcomm_Dragonwing_RB3_Gen2_Vision_Development_Kit_Release_Notes.pdf">release notes</a>
+        </li>
+      </ul>
+      <hr class="p-rule--muted" />
+      <div class="p-notification--information is-borderless">
+        <div class="p-notification__content">
+          <p class="p-notification__message">
+            The Ubuntu images are tested on the <a href="https://www.qualcomm.com/developer/hardware/rb3-gen-2-development-kit">Qualcomm® RB3 Gen 2 dev kit</a>
```

Comment:
```suggestion
            The Ubuntu images are tested on the <a href="https://www.qualcomm.com/developer/hardware/rb3-gen-2-development-kit">Qualcomm® RB3 Gen 2 dev kit</a>
```
A more description aria-label would be nice here